browser/src/control/Control.Zotero.js

```diff
@@ -196,6 +196,32 @@ L.Control.Zotero = L.Control.extend({
 		};
 	},
 
+	getCitationJSONString: function(item) {
+		var resultJSON = {};
+		resultJSON['citationID'] = Math.round(Date.now() / 1000).toString();
```

Comment:
This does not look like a random string. We need a random string, 10 characters long, with characters from the [a-zA-Z0-9] range.

Reply:
updated